internal/clock/clock.go

```diff
@@ -44,12 +48,19 @@ func NewMock() *Mock {
 
 // Add moves the current time of the mock clock forward by the duration.
 // This should only be called from a single goroutine at a time.
-func (m *Mock) Add(d time.Duration) {
+func (m *Mock) Add(d time.Duration, condition *gopeek.Condition) {
 	m.Lock()
 	// Calculate the final time.
 	end := m.now.Add(d)
 
 	for len(m.timers) > 0 && m.now.Before(end) {
+		// Wait before popping from timers
```

Comment:
Without nap ( Strictly speaking the current test can fail even with nap if a gorotine invoking time.Sleep wake up before workers start to run. actually I've faced sometimes while testing ) 
there is a possibility that 
```go
t := heap.Pop(&m.timers).(*Timer)
```
may be invoked before one of workers running job get into the point [clock.Sleep](https://github.com/uber-go/ratelimit/blob/master/ratelimit.go#L121) .
in this case, as there are still only timers registered by clock.AfterFunc , test will fail.